Makefile

```diff
@@ -46,12 +46,13 @@ all: build
 help: ## Display this help.
 	@awk 'BEGIN {FS = ":.*##"; printf "\nUsage:\n  make \033[36m<target>\033[0m\n"} /^[a-zA-Z_0-9-]+:.*?##/ { printf "  \033[36m%-15s\033[0m %s\n", $$1, $$2 } /^##@/ { printf "\n\033[1m%s\033[0m\n", substr($$0, 5) } ' $(MAKEFILE_LIST)
 
-kind-create: ## Create a kind cluster named ${KIND_CLUSTER_NAME} locally if necessary
+kind-create: ## Create a kind cluster named ${KIND_CLUSTER_NAME} locally if necessary and save the kubectl config.
 ifeq (1, $(shell kind get clusters | grep ${KIND_CLUSTER_NAME} | wc -l | tr -d ' '))
 	@echo "Cluster already exists"
 else
 	@echo "Creating Cluster"
 	kind create cluster --name ${KIND_CLUSTER_NAME} --image=kindest/node:${K8S_NODE_IMAGE} --config ${KIND_CONFIG_DIR}/cluster.yaml
+	cp -r ${HOME}/.kube .
 endif
```

Comment:
There is no problem with cp command running in the dev container, but I'm not happy if it runs locally.
Can you make `if` depending on whether it's in the dev container?

i think, dev container can set some environment variables from `devcontainer.json`.
ex. Create some check variable like `IN_DEV_CONTAINER` and check it.
<https://code.visualstudio.com/remote/advancedcontainers/environment-variables>

Reply:
@itiB 
I added `IN_DEV_CONTAINER` to check whether it's in the dev container.
Thank you for your comments.

Reply:
`cp` command creates a `.kube` directory under the existing `.kube` directory, so I changed it to use `kind get kubeconfig`.